public/styles/style.css

```diff
@@ -1,3 +1,106 @@
+html {
```

Comment:
@des-des we don't know what you mean, we haven't touched style.css, it's being compiled with sass...